README.md

```diff
@@ -8,58 +8,72 @@ appropriate helm command based on which action it is included within: `install`,
 
 ### Install or Upgrade
 
-```
+```shell
 porter mixin install helm
 ```
 
+### Configure mixin repositories
+
+```yaml
+- helm:
+  repositories:
```

Comment:
Technically, I think this `repositories` line needs to be indented 2 spaces (so, the rest of the section as well...)  Otherwise, I seem to be getting:
```
$ porter build
Error: error unmarshaling the typed manifest: mixin declaration contained more than one mixin
```